src/utils/plugins.ts

```diff
@@ -0,0 +1,24 @@
+import { FastifyRequest } from 'fastify'
+
+export function generateForwardHeaders(
```

Comment:
You can move this function inside the `plugins/postgrest.ts` instead of creating a new file